Firestore/core/test/unit/remote/grpc_connection_test.cc

```diff
@@ -125,8 +126,9 @@ TEST_F(GrpcConnectionTest, GrpcStreamingCallsNoticeChangeInConnectivity) {
   std::unique_ptr<GrpcStreamingReader> streaming_call =
       tester.CreateStreamingReader();
   streaming_call->Start(
-      [&](const StatusOr<std::vector<grpc::ByteBuffer>>& result) {
-        if (IsConnectivityChange(result.status())) {
+      static_cast<size_t>(0), [&](const std::vector<ResponsesT>) {},
```

Comment:
There are a lot of `static_cast<size_t>(...)` instances in the PR (for both `Start` and `StartReader`). Are you sure these are necessary? I thought the compiler should be able to implicitly cast a literal number argument to `size_t`.

Reply:
I think the compiler will do the implicitly cast, but I want to make the code less dependents on compiler behaviour :( Is it really a bad idea to increase code size a little bit?

Reply:
These casts do not increase code size (they get optimized away by the compiler). It's more about readability. I think the code becomes much easier to read without the `static_cast`s. In this case I think all compilers should behave the same.